core/modules/util/EventThread.h

```diff
@@ -153,7 +153,8 @@ class PoolEventThread : public EventThread {
 class ThreadPool : public std::enable_shared_from_this<ThreadPool> {
 public:
     using Ptr = std::shared_ptr<ThreadPool>;
-    static ThreadPool::Ptr newThreadPool(uint thrdCount, CommandQueue::Ptr const& q) {
+    static ThreadPool::Ptr newThreadPool(unsigned int thrdCount,
+					 CommandQueue::Ptr const& q) {
```

Comment:
You're right.  My emacs is still sticking in tabs; I replaced them with spaces and all is well.